### ISSUE TITLE
Automatically adjust text color on light backgrounds

### ISSUE THREAD

**Issue by samsledje** [samsledje/markdown-media-tracker#28]
If the user selects a primary color that violates the WCAG contrast ratio test, fonts should be darkened or lightened so that there is appropriate contrast. Do this in a way that is compute efficient, and only checks once on the user save of a new primary color. Be sure that you are testing different font colors separately-- for example, the book title and author have different text colors, so should be adjusted separately.

**Comment by samsledje** [comment]
Current PR is #38
